packages/apps/transfer/jest.config.js

```diff
@@ -3,7 +3,7 @@ const createJestConfig = nextJest({
   dir: './',
 });
 const customJestConfig = {
-  reporters: ['jest-standard-reporter'],
+  reporters: ["default", "jest-junit"],
```

Comment:
jest-standard-reporter has not been updated since 2020, so I used junit package which is new and recommended by jest